modules/migration-migrating-on-prem-to-cloud.adoc

```diff
@@ -0,0 +1,104 @@
+* The `crane` utility is installed. (??? Need to add a topic on how to install it. ???)
+* The MTC web console and Migration Controller must be installed on the target cluster.
+
+Replace ${namespace} with either openvpn or the namespace you specified when running the command to set up the tunnel.
+
+.Procedure
+
+// ??? "Command system" is a placeholder for the system issuing the `tunnel-api` command. ???
+
+. Log in remotely to a node on the source cluster and a node on the destination cluster
+
+. Obtain the cluster-context for both clusters after logging in. ???HOW???
+
+. Establish a tunnel by entering the following command on the command system:
++
+[source,terminal]
+----
+$ crane tunnel-api [--namespace _<namespace>_] \
+      --destination-context _<destinatin-cluster>_ \
+      --source-context _<source-cluster>_
+----
++
+If you don't specify a namespace, the command uses the default value `openvpn`.
+
+For example:
++
+[source,terminal]
+----
+$ crane tunnel-api --namespace my_tunnel \
+      --destination-context openshift-migration/c131-e-us-east-containers-cloud-ibm-com/admin \
+      --source-context default/192-168-122-171-nip-io:8443/admin
+----
++
+The command generates SSL Certificates. This process might take several minutes. A message appears when the process completes.
++
+The OpenVPN server starts on the destination cluster and the OpenVPN client starts on the source cluster.
++
+After a few minutes, the load balancer resolves on the source node.
+[TIP]
+====
+You can view the log (???WHICH LOG?) to check the status of this process by entering the following commands with root privileges (???WHAT ExACTLY ARE THESE COMMANDS? `oc get po` is not in the CLI reference):
+
+[source,terminal]
+----
+# oc get po -n _<namespace>_
+NAME            READY     STATUS      RESTARTS    AGE
+???What should go here???
+#oc logs -f -n _<namespace>_ _<namespace_id>_ -c openvpn
+----
+When the address of the load balancer is resolved, the message `Initialization Sequence Completed` appears at the end of the log.
+====
+
+. On the openvpn server, which is on a destination control node, verify that the `openvpn` service and the `proxied-cluster` service are running:
++
+[source,terminal]
+----
+$ oc get service -n _<namespace>_
+----
+
+. On the source node, get the SA token for the migration controller:
++
+[source,terminal]
+----
+# oc sa get-token -n openshift-migration migration-controller
+----
+
+. Launch the {mtc-short} web console and add the destination cluster, using the following values:
++
+* *Cluster name*: The destination namespace.
+* *URL*: `proxied-cluster._<namespace>_`.svc.cluster.local
+* *Service account token*: The token from the source node.
```

Comment:
= token of the migration controller service account.

Reply:
Fixed.